### PR TITLE
[ZEPPELIN-5792][FRONT-BUILD] Remove bower

### PR DESCRIPTION
**Pull request by holodazoltan**
### What is this PR for?
Remove deprecated bower. As the "bower" says, recommended using a different system. So it looks good to remove bower. Replace bower by npm.

### What type of PR is it?
Improvement

### Todos
* [x] - Bower's dependencies are resolved by npm
* [x] - Minor dependency versions may have been changed to resolved by npm
* [x] - Integrate new dependencies to front-end and testing

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-5792

### How should this be tested?
* Run npm commands
* Run Karma tests
* Manual front-end testing

### Questions:
* Does the license files need to update? Maybe because of library version change
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by jongyoul** [comment]
@holodazoltan Hello ~ Thank you for the outstanding contribution. I'm not an expert on the front but I heard that bower is outdated and should be changed to others.

BTW, I have one minor question. Should we change yarn to do it? We already are using npm and yarn is similar to npm. I basically, would like to keep the current way if it doesn't have any critical issues. I'm not saying which one is better or not, nor which one is promising but I would like to know if we can solve this problem with npm as is. Could you please give me your opinion on it?

**Comment by jongyoul** [comment]
@holodazoltan Thank you for the reply and for making the changes. I look forward to seeing it.

**Comment by jongyoul** [comment]
@holodazoltan Thank you for the contribution. Let's see how CI goes.

**Comment by holodazoltan** [comment]
@jongyoul CI was not ok. In local all zeppelin-web e2e test cases run fine, I think there is some config issue here, CI env has many timeout cases. I pushed a new commit based on protactor issue [suggestions](https://github.com/angular/protractor/issues/4697).

**Comment by Reamer** [comment]
I recently added two PR (https://github.com/apache/zeppelin/pull/4638 , https://github.com/apache/zeppelin/pull/4637) for a more stable CI build. Could you please do a rebase so we can better look at where the CI job breaks.

File: zeppelin-web/src/index.html [hunk 1]
```diff
@@ -36,19 +36,18 @@
...
+    <link rel="stylesheet" href="node_modules/nvd3/build/nv.d3.css" />
+    <link rel="stylesheet" href="node_modules/perfect-scrollbar/src/perfect-scrollbar.css" />
+    <link rel="stylesheet" href="node_modules/ng-sortable/dist/ng-sortable.css" />
+    <link rel="stylesheet" href="node_modules/angular-xeditable/dist/css/xeditable.css" />
+    <link rel="stylesheet" href="node_modules/highlight.js/styles/github.css" />
+    <link rel="stylesheet" href="node_modules/ng-toast/dist/ngToast.css" />
+    <link rel="stylesheet" href="node_modules/bootstrap3-dialog/dist/css/bootstrap-dialog.min.css" />
+    <link rel="stylesheet" href="node_modules/jquery-ui/themes/base/all.css" />
+    <link rel="stylesheet" href="node_modules/select2/dist/css/select2.css" />
+    <!-- endnpm -->
```
Comment (matthias-koch) [inline zeppelin-web/src/index.html:50]:
```suggestion
    <!-- endnpm -->
    <!-- endbuild -->
```
An endbuild is missing here.

**Comment by jongyoul** [comment]
@holodazoltan Hello, is there any update or problem? I believe that your PR is really helpful for the community but it should pass CI anyway. Can you rebase it to the current master as @Reamer suggested? I can help you if you have any critical issue.

**Comment by matthias-koch** [comment]
It appears that @holodazoltan has ceased reporting. If there are no objection, I would like to resubmit the PR with my bug fix.

**Comment by Reamer** [comment]
> It appears that @holodazoltan has ceased reporting. If there are no objection, I would like to resubmit the PR with my bug fix.

Gladly, but please include holodazoltan as a co-author.

**Comment by holodazoltan** [comment]
Sorry for late reply, my change has some CSS bundling issue, I was working on that, but I don't have any solid result yet. CSS issue can be easily reproduced in local, but I was testing e2e with docker image (old site, not updated one) :(

[matthias-koch](https://github.com/matthias-koch) [Reamer](https://github.com/Reamer) Feel free to resuse and submit any part, I am glad if we can resolve this issue.

**Comment by Reamer** [comment]
Closed because follow-up pull request #4651 was merged.

**Pull request closed without merging**